docs/source/examples/plots.rst

```diff
@@ -84,7 +79,6 @@ Examples to illustrate the use of :class:`.GraphScene` in manim.
 
 
 .. manim:: Plot4SinCos
-    :quality: medium
     :save_last_frame:
 
     class Plot4SinCos(GraphScene):
```

Comment:
I don't think that others are too simple.
In case that you look for something specific, its much better to have lots of simple examples then one difficult one.
And I think especially beginners profit a lot from this simple to complex structure that I try to follow